lighthouse-cli/performance-experiment/server.js

```diff
@@ -30,7 +30,7 @@ const http = require('http');
 const parse = require('url').parse;
 const opn = require('opn');
 const log = require('../../lighthouse-core/lib/log');
-const reportGenerator = new (require('./report/perf-x-report-generator'))();
+const ReportGenerator = require('./report/perf-x-report-generator');
```

Comment:
nit: maybe call this `PerfXReportGenerator` just to more clearly call it out as an instance of the subclass?